lib/src/Uncertainty/Algorithm/OrthogonalBasis/OrthogonalFunctionFactory.cxx

```diff
@@ -62,6 +62,20 @@ Function OrthogonalFunctionFactory::build(const UnsignedInteger ) const
 }
 
 
+/* Build the Function of the given multi-indices */
+Function OrthogonalFunctionFactory::build(const Indices & indices) const
+{
+  try
```

Comment:
Maybe we should check dimension of indices before otherwise it is ok for me

Reply:
The check should be present in inverse function, ok for Linear but I see it's missing in Hyperbolic, so I'd add it there.

Reply:
ok

Reply:
done